### ISSUE TITLE
Dataloader selection problem.

### ISSUE THREAD

**Issue by jinyuan30** [yassouali/pytorch_segmentation#33]
Hi.
I have a my own custom dataset that contains images and coresponding masks, and there are five classes.
How can I select type of train_loader?
I saw this repository give four kind of dataloader but I don't know how to select a proper one.

**Comment by jinyuan30** [comment]
The five label values in the mask are 0, 1, 2, 3, 4, by the way.

**Comment by yassouali** [comment]
You'll need to create your own, see the closed issues, you'll find similar questions.
